python/ray/serve/benchmarks/cluster.yaml

```diff
@@ -27,7 +27,7 @@ worker_nodes:
 initialization_commands: []
 setup_commands:
     - sudo apt-get install build-essential libssl-dev git -y
-    - 'rm -r wrk || true && git clone https://github.com/wg/wrk.git wrk && cd wrk && make -j && cp wrk /usr/local/bin'
+    - 'rm -r wrk || true && git clone https://github.com/wg/wrk.git /tmp/wrk && cd /tmp/wrk && make -j && sudo cp wrk /usr/local/bin'
```

Comment:
shouldn't this be
```
sudo cp /tmp/wrk /usr/loca/bin
```

Reply:
nope we are coping the binary so it's actually `/tmp/wrk/wrk` but we cd there already

Reply:
Ahh okay!